build/source/driver/summa_defineOutput.f90

```diff
@@ -101,11 +100,13 @@ subroutine summa_defineOutputFiles(modelTimeStep, summa1_struc, err, message)
  ! *** define the name of the model output file
  ! *****************************************************************************
 
- ! define name of output file : spinup
+ ! define full name of output file 
  if(modelTimeStep==1)then
   select case(newOutputFile)
-   case(noNewFiles);       fileout = trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//'output'//trim(output_fileSuffix)
-   case(newFileEveryOct1); fileout = trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//'spinup'//trim(output_fileSuffix)
+   !case(noNewFiles);       fileout = trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//'output'//trim(output_fileSuffix)
+   case(noNewFiles);       fileout = trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//trim(output_fileSuffix)
+   !case(newFileEveryOct1); fileout = trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//'spinup'//trim(output_fileSuffix)
+   case(newFileEveryOct1); fileout = trim(OUTPUT_PATH)//trim(OUTPUT_PREFIX)//trim(output_fileSuffix)
    case default; err=20; message=trim(message)//'unable to identify the option to define new output files'; return
   end select
```

Comment:
It would be cleaner to remove commented lines

Reply:
right, thanks - I left them in in case you advised not to make the changes, in which case it would be easier for me to revert them.  I was planning to remove them in a clean PR after making suggested changes.  but I can see this is a little messy.